src/Tribe/Ignored_Events.php

```diff
@@ -626,7 +618,7 @@ public function register_ignored_post_status() {
 		 *
 		 * @return null|bool
 		 */
-		public function action_pre_delete_event( $check, $post, $force_delete ) {
+		public function action_pre_delete_event( $unused_check, $post, $force_delete ) {
```

Comment:
you need to rename the variable to make TribalScents pass if you aren't going to use it